src/README.md

```diff
@@ -0,0 +1,499 @@
+# Maynard
+
+Functionality to generate transition matrices, absorbing states, and other
+results for the heterogeneous moran process.
+
+## Tutorial
+
+In this tutorial we will see how to use `maynard` in order to generate the
+transition matrix defining a heterogeneous public goods game. For background
+information on Markov processes in general we recommend:
+Probability, Markov Chains, Queues, and Simulation by
+William J Stewart
```

Comment:
```suggestion
information on Markov processes in see [1].
```

Then include bibliographic information at the end (like in a LaTeX document althgouh you have to do it manually). But you should be precise with the information:

```
Stewart, William J. Probability, Markov chains, queues, and simulation: the mathematical basis of performance modeling. Princeton university press, 2009.
```

You do also need to point at a reference for the Moran process here.